tests/test_shotgun_model_data_handler.py

```diff
@@ -0,0 +1,144 @@
+# Copyright (c) 2013 Shotgun Software Inc.
```

Comment:
I had totally forgotten that i had started the tests a while back. Thanks for reminding. There is now a half-decent test rig in place for shotgun utils (yay!). It's light weight and can be a recipe for other frameworks to implement our standard test rig. It ended up being pretty neat actually :)

Tests added for the low level data handling.